### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by rabayarabu**
- Use React components.
- Use React props.
- Use React state.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function Calculator(property) {
+  const { name, className } = property;
+  return (
+    <button className={className} type="submit">{name}</button>
+  );
+}
```
Comment (Ismailco) [inline src/components/Calculator.js]: - [ ] The `Calculator.js` should have at least 1 child component, kindly make sure to add a child component for the Calculator

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import './index.css';
+import CalculatorDesign from './components/CalculatorDesign';
```
Comment (Ismailco) [inline src/App.js]: - [ ] You can import this component (`CalculatorDesign`) inside the `Calculateor` component and then import the `Calculator` component in the `App.js`, kindly make sure to import the `Calculator` component here instead

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @rabayarabu, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Create the UI as requested
✔️ Used CRA
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @rabayarabu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Request Changes ✔️
- Well Structured Code ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**